api/app/controllers/spree/api/v2/storefront/account/addresses_controller.rb

```diff
@@ -0,0 +1,70 @@
+module Spree
+  module Api
+    module V2
+      module Storefront
+        module Account
+          class AddressesController < ::Spree::Api::V2::BaseController
+            before_action :require_spree_current_user
+
+            def index
+              render_serialized_payload { serialize_collection(collection) }
+            end
+
+            def create
+              resource = collection.new(address_params)
```

Comment:
@kgorazd let's move this into `Account::Addresses::Create` service

Reply:
👍

Reply:
fixed